objectherkenning_openbare_ruimte/data_sampling/decos_helper.py

```diff
@@ -0,0 +1,97 @@
+import datetime
+import pathlib
+
+import geopandas as gpd
+import pandas as pd
+
+RD_CRS = "EPSG:28992"  # CRS code for the Dutch Rijksdriehoek coordinate system
+
+
+def _decos_df2gdf(decos_df: pd.DataFrame) -> gpd.GeoDataFrame:
+    """
+    Convert Decos DataFrame to GeoDataFrame. Geometry will be parsed from WKT string. The columns
+    datum_object_van and datum_object_tm will be converted to datetime object for later use.
+
+    Parameters
+    ----------
+    decos_df: DataFrame
+        The DataFrame to convert
+
+    Returns
+    -------
+    GeoDataFrame with Decos data
+    """
+    decos_gdf = gpd.GeoDataFrame(
+        decos_df,
+        geometry=gpd.GeoSeries.from_wkb(decos_df.geometrie_locatie),
+        crs=RD_CRS,
+    )
+    decos_gdf["datum_object_van"] = pd.to_datetime(
+        decos_gdf["datum_object_van"], format="%Y-%m-%d"
+    )
+    decos_gdf["datum_object_tm"] = pd.to_datetime(
+        decos_gdf["datum_object_tm"], format="%Y-%m-%d"
+    )
+    return decos_gdf
+
+
+def load_decos_gdf(path: str) -> gpd.GeoDataFrame:
```

Comment:
What is the advantage of having this function wrapping the other? Cannot we use just one?

Reply:
The wrapper is a left-over from development, I actually don't use it anywehere I think. Will simply delete it.